lib/vehicle_factory.rb

```diff
@@ -0,0 +1,27 @@
+class VehicleFactory
+              
+  def initialize
+  end
```

Comment:
If we aren't doing anything in our initialize method, we can actually just get rid of it! Ruby knows how to initialize if there's nothing else that needs to happen out of the ordinary.
```suggestion
```